vms/platformvm/txs/executor/camino_helpers_test.go

```diff
@@ -31,21 +31,22 @@ import (
 	"github.com/ava-labs/avalanchego/vms/platformvm/api"
 	"github.com/ava-labs/avalanchego/vms/platformvm/config"
 	"github.com/ava-labs/avalanchego/vms/platformvm/genesis"
+	"github.com/ava-labs/avalanchego/vms/platformvm/locked"
 	"github.com/ava-labs/avalanchego/vms/platformvm/metrics"
 	"github.com/ava-labs/avalanchego/vms/platformvm/reward"
+	"github.com/ava-labs/avalanchego/vms/platformvm/stakeable"
 	"github.com/ava-labs/avalanchego/vms/platformvm/state"
 	"github.com/ava-labs/avalanchego/vms/platformvm/status"
 	"github.com/ava-labs/avalanchego/vms/platformvm/txs"
 	"github.com/ava-labs/avalanchego/vms/platformvm/txs/builder"
 	"github.com/ava-labs/avalanchego/vms/platformvm/utxo"
+	"github.com/ava-labs/avalanchego/vms/secp256k1fx"
 )
 
-const defaultCaminoValidatorWeight = 2 * units.KiloAvax
```

Comment:
whats reason to revert it?

Reply:
Because in genesis the initial UTXOs are 500 MilliAvax so if the weight was 2 KiloAvax the tx would always fail with "not enough balance to lock"

Reply:
well, u could also use another value in genesis - you' generating it in buildCaminoGenesisTest func with your own values anyway, just replace avax ones
but I'm also fine with reverting it, cause it doesn't matter what values are used in tests if they valid in conjunction

Reply:
genesis_test (the allocation) is used for all tests (camino + avax). So to get it work, we have to set the const `defaultCaminoValidatorWeight = 2 * units.KiloAvax` to a lower value, e.g. `defaultCaminoValidatorWeight = 10 * units.MilliAvax`.
Our tests should be performed with Min = Max.

Reply:
I left that changes in a separate commit to be more readable @c4t-ag

Reply:
@c4t-ag yep, we just discussed it in discord, that min should be equal to max, forgot to mention it that, when wrote last comment

Reply:
@c4t-ag in last push all camino helpers has 2 kiloAvax with min == max and ava helpers are unchanged, so it should be fine

Reply:
Pls. squash the 2 commits